src/views/Article/Index.jsx

```diff
@@ -37,10 +38,13 @@ import {
 import Footer from '../../components/footer/Index';
 import Header from '../../components/header/Index';
 import Loader from '../../components/loader/Index';
+import Button from '../../components/button/Index';
 import './article.scss';
 import Comment from '../../components/comment/Index';
 import ViewComment from '../../components/viewComment/Index';
 import ReportModal from '../../components/reportArticle/Index';
+import OptionModal from '../../components/modal';
```

Comment:
Let's change the folder name of this component to `OptionModal`. It's not the only modal we have in the app.

Reply:
ok.